code/game/machinery/computer/abnormality_work.dm

```diff
@@ -21,6 +21,10 @@
 	var/obj/machinery/containment_panel/linked_panel
 	/// Accumulated abnormality chemical.
 	var/chem_charges = 0
+	/// Abnormality name override
+	var/abno_name
+	/// Abnormality risk level override
+	var/abno_threat_level
```

Comment:
Army in Black:
~Kirie said no~

Reply:
This is intended for work mechanics, like with DF and pinocchio.

Reply:
Ye ye, just saying big sad no apply to AIB, truly big said yes yes